### PR TITLE
docs: Add TechDocs troubleshooting

### PR DESCRIPTION
**Pull request by adamdmharvey**
## Hey, I just made a Pull Request!

Based on some searches on Discord and local testing, a quick troubleshooting section for how to use TechDocs CLI locally.

Welcome adjustments/tweaks to any assumptions I made!

image

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)

### CODE REVIEW

File: docs/features/techdocs/troubleshooting.md [hunk 1]
```diff
@@ -5,6 +5,53 @@ sidebar_label: Troubleshooting
...
+
+TechDocs will fail to clone your docs if you have a git config which overrides
+the `https` protocol with `ssh` or something else. Make sure to remove your git
+config locally when you try TechDocs.
+
+## MkDocs Build Errors
+
+Using the [TechDocs CLI](https://github.com/backstage/techdocs-cli), you can
+troubleshoot MkDocs build issues locally. Note this requires you have Docker
+available to launch images. First, `git clone` the target repository locally,
```
Comment (OrkoHunter) [inline docs/features/techdocs/troubleshooting.md:18]: Soon it will have a `--no-docker`. :)

**Review by OrkoHunter — APPROVED** [review]
Thanks! :shipit:

File: docs/features/techdocs/troubleshooting.md (hunk 1 above)
Comment (adamdmharvey) [reply docs/features/techdocs/troubleshooting.md:18]: shipit + iterate, eh?

**Comment by adamdmharvey** [comment]
PS: i thought you already fixed that project board GItHub token for forks issue? 😆

File: docs/features/techdocs/troubleshooting.md (hunk 1 above)
Comment (OrkoHunter) [reply docs/features/techdocs/troubleshooting.md:18]: Yep yep

**Comment by OrkoHunter** [comment]
> PS: i thought you already fixed that project board GItHub token for forks issue? 😆

Indeed, this seems strange. I added [these checks](https://github.com/adamdmharvey/backstage/blob/33f7fbc5b20c8514eab2931555b125cebba8bd29/.github/workflows/techdocs-project-board.yml#L42) to not run the workflow when the token is missing (which happens on a PR from fork).

**Pull request merged**